WordPress/wordpress-playground
# Website: Support Base64-encoding Blueprints passed in the URL

Blueprints contain many symbols that need to be URLencoded and posting complex Blueprints on sites like GitHub poses double encoding and formatting challenges. This PR adds support for Base64-encoded Blueprints.

## Testing instructions

Go to
http://localhost:5400/website-server/#eyJzaXRlT3B0aW9ucyI6eyJibG9nbmFtZSI6IkJhc2U2NCBlbmNvZGluZyB3b3JrcyJ9fQ== and confirm the site title says "Base64 encoding works"
